website/docs/about/support.md

```diff
@@ -28,4 +28,8 @@ Users external to HashiCorp can [file issues in GitHub](https://github.com/hashi
 
 ## Office hours
 
-Office hours are a great way to get feedback on your project or help to get unstuck in a code migration. [Sign up for an appointment](https://docs.google.com/document/d/1LdyQznltt4VCbGGGDuDkARRRKXI4laCHxHhY8dJy4wE/edit?usp=sharing) at the next office hours.
\ No newline at end of file
+Office hours are a great way to get feedback on your project or help to get unstuck in a code migration. [Sign up for an appointment](https://docs.google.com/document/d/1LdyQznltt4VCbGGGDuDkARRRKXI4laCHxHhY8dJy4wE/edit?usp=sharing) at the next office hours.
+
+## Helios Feedback
```

Comment:
```suggestion
## Share your feedback
```

or we could consider even `Share your feedback anonymously` which it's even more explicit.

Reply:
the anonymous aspect is referenced in the copy below but I did update this to `Share Feedback`